src/components/dashboard/FaceVerification/sdk/FaceTecGlobalState.js

```diff
@@ -16,22 +15,19 @@ const FaceTecGlobalState = {
    * Convenience method to initialize the FaceTec SDK.
    */
   async initialize() {
-    let { faceTecSDKInitializing } = this
     const { faceTecLicenseKey, faceTecLicenseText, faceTecEncryptionKey } = Config
 
-    if (!faceTecSDKInitializing) {
+    if (!this.faceTecSDKInitializing) {
```

Comment:
why this was needed ?

Reply:
its just style refactor.
in my opinion the destructor/assign pattern is not readable.
`this.` marks it clearly as a class variable

Reply:
okay, i think we just need to discuss and agree with some unified coding style for the whole project
probably we could do it after we release Android version and solve current bug ?